drivers/DeepSmart/deepsmart/src/lifecycles.lua

```diff
@@ -0,0 +1,83 @@
+local config = require('config')
+local log = require('log')
+local caps = require('st.capabilities')
+local wisers = require('deepsmart.wisers')
+local lifecycle_handler = {}
+
+function lifecycle_handler.init(driver, device)
+  -- report test data
+  --wisers.default_report(driver, device)
+  local is_bridge = wisers.is_device_bridge(device)
+  log.info('device '..device.id..' in init')
+  -- for bridge do not reload devices&&config
+  -- just use the devices&&config loaded from wiser's datastore
+  -- only discover or refresh bridge will reload devices&&config
+  if (is_bridge) then
+    -- just add wiser(bridge) to wisers
+    wisers.add_wiser(device.device_network_id, device:get_field(config.FIELD.IP), device)
```

Comment:
You have to refresh your wiser again othersize your edge devices can't be added
```
wisers.refresh_wiser(device.device_network_id)
```

Reply:
add refresh_wiser